samples/cmake/FindOpenCL.cmake

```diff
@@ -1,3 +1,19 @@
+# ########################################################################
+# Copyright 2015 Advanced Micro Devices, Inc.
+# 
```

Comment:
According to blame file it should be Vratis but when I looked at the file I agree that your modifications related to CL_VERSION_2_0 are crucial so AMD is more convenient.